src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageMetadataProvider.cs

```diff
@@ -212,7 +212,7 @@ private static async Task<PackageDeprecationMetadata> MergeDeprecationMetadataAs
         private static IEnumerable<PackageVulnerabilityMetadata> MergeVulnerabilityMetadata(PackageIdentity identity, IEnumerable<IPackageSearchMetadata> packages)
```

Comment:
`identity` parameter is unused in this method.

Reply:
@jebriede this was marked resolved but hasn't been changed. What's the reasoning for keeping the parameter?

Reply:
It has been changed and I'll push the update in batch with other changes. There's no reason to keep this parameter and it was part of a method that existed but I've cleaned it up based on Kartheek's suggestion.